### ISSUE TITLE
PR #2751 breaks DWAV

### ISSUE THREAD

**Issue by gold2718** [ESMCI/cime#2764]
PR #2751 introduced a new method for detecting a post-data-assimilation run, however, DWAV still uses the old method and no longer compiles.

**Comment by gold2718** [comment]
@jedwards4b, I thought we were going to hold off on merging #2751 until we had branch tags for CAM and CLM. I created #2759 so I could test these branches but did not expect the #2751 merge. What is the best approach here? Just do a quick fix for DWAV? I can test DWAV on Hobart, do I also have to wait until Cheyenne is back up to run scripts_regression_tests?

**Comment by jedwards4b** [comment]
My understanding from yesterdays cseg meeting was that those tags were imminent.   This PR indicates that scripts_regression_tests.py was run, since that includes tests with DWAV I would have expected that failure to be reported.

**Comment by gold2718** [comment]
Summary from phone call: There is no DWAV test in scripts_regression_tests.

**Comment by ekluzek** [comment]
@jedwards4b in cseg we talked about the fact that we created an alpha tag to put tags for this in. But, literally right now, there are no tags listed in that alpha tag. I envisioned that we would start with branches for cime, cam, and clm, and eventually those branches would become tags on master. But, since there were no tags listed, I'm not sure how that became "those tags were imminent". In any case, I've started the branch for ctsm. And I'll let the two of you decide how to handle this in cime. Is the branch for CAM started?
